src/main/java/com/projetoGerenciamentoCurso/gerenciamentoCurso/controller/AlunoController.java

```diff
@@ -14,17 +14,11 @@
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.dto.AlunoDto;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.models.Aluno;
 import com.projetoGerenciamentoCurso.gerenciamentoCurso.service.AlunoService;
-import com.projetoGerenciamentoCurso.gerenciamentoCurso.service.repositoryServices.AlunoRepositoryService;
-
-
 
 @RestController
 @RequestMapping(path = AlunoController.PATH)

```

Comment:
Adicionar a anotação `@AllArgsConstructor` para seguir o padrão descrito na documentação https://tdn.totvs.com.br/pages/viewpage.action?pageId=514721534#Defini%C3%A7%C3%B5es-Back-end-Utilizeinje%C3%A7%C3%A3odedepend%C3%AAncia(aka.DI-DependencyInjection)atrav%C3%A9sdeconstrutores
```suggestion
@RequestMapping(path = AlunoController.PATH)
@AllArgsConstructor
```

Reply:
feito